eventsources/common/aws/aws_test.go

```diff
@@ -68,7 +68,7 @@ func TestAWS(t *testing.T) {
 	})
 
 	convey.Convey("Given kubernetes secret that hold credentials, create AWS credential", t, func() {
-		secret, err := client.CoreV1().Secrets(namespace).Create(context.Background(), &corev1.Secret{
+		secret, err := client.CoreV1().Secrets(namespace).Create(context.TODO(), &corev1.Secret{
```

Comment:
I think that `Background` might be fine here. Practically `Background` and `TODO` are the same - empty contexts, but semantically they are different. I believe you should use `TODO` when you think this will need to changed in the future. If this is not the case, I think it is fine to leave it.